axelor-account/src/main/java/com/axelor/apps/account/service/invoice/generator/InvoiceGenerator.java

```diff
@@ -313,6 +313,14 @@ protected Invoice createInvoiceHeader() throws AxelorException {
 
     invoice.setInvoicesCopySelect(getInvoiceCopy());
 
+    // PFP
+    if (invoice.getCompany().getAccountConfig().getIsManagePassedForPayment()
```

Comment:
Please instead of calling `invoice.getCompany().getAccountConfig()` use the existing `accountConfig` variable above